spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/mapping/AssociationLinks.java

```diff
@@ -99,4 +103,31 @@ public boolean isLinkableAssociation(PersistentProperty<?> property) {
 		metadata = mappings.getMappingFor(property.getActualType());
 		return metadata == null ? false : metadata.isExported();
 	}
+
+	/**
+	 * Returns whether the given property is an association that should be exported.
+	 *
+	 * @param property can be {@literal null}
+	 * @return
+	 */
+	public boolean isExportableAssociation(PersistentProperty<?> property) {
```

Comment:
I briefly thought about this since I was going to simply add this logic to the existing isLinkable() function. But then I was worried it might break existing semantics and decided to split it up. I can do some more investigation.